DnD Character Track build issues

Hey everyone, 

I am currently going through the Java track and I just downloaded the exercise. 
This exercise gives me a lot of errors but that is without me implementing anything. It looks like the solution .java file has a lot of **missing methods** and that is uncommon for the track. 

Here is the error log. 

```
> Executing task: gradle test <


> Task :compileTestJava FAILED
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:118: error: cannot find symbol
            assertTrue(character.getDexterity() > 2 && character.getDexterity() < 19);
                                ^
  symbol:   method getDexterity()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:118: error: cannot find symbol
            assertTrue(character.getDexterity() > 2 && character.getDexterity() < 19);
                                                                ^
  symbol:   method getDexterity()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:119: error: cannot find symbol
            assertTrue(character.getConstitution() > 2 && character.getConstitution() < 19);
                                ^
  symbol:   method getConstitution()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:119: error: cannot find symbol
            assertTrue(character.getConstitution() > 2 && character.getConstitution() < 19);
                                                                   ^
  symbol:   method getConstitution()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:120: error: cannot find symbol
            assertTrue(character.getIntelligence() > 2 && character.getIntelligence() < 19);
                                ^
  symbol:   method getIntelligence()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:120: error: cannot find symbol
            assertTrue(character.getIntelligence() > 2 && character.getIntelligence() < 19);
                                                                   ^
  symbol:   method getIntelligence()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:121: error: cannot find symbol
            assertTrue(character.getWisdom() > 2 && character.getWisdom() < 19);
                                ^
  symbol:   method getWisdom()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:121: error: cannot find symbol
            assertTrue(character.getWisdom() > 2 && character.getWisdom() < 19);
                                                             ^
  symbol:   method getWisdom()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:122: error: cannot find symbol
            assertTrue(character.getCharisma() > 2 && character.getCharisma() < 19);
                                ^
  symbol:   method getCharisma()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:122: error: cannot find symbol
            assertTrue(character.getCharisma() > 2 && character.getCharisma() < 19);
                                                               ^
  symbol:   method getCharisma()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:123: error: cannot find symbol
            assertEquals(character.getHitpoints(),
                                  ^
  symbol:   method getHitpoints()
  location: variable character of type DnDCharacter
C:\Users\Mayur\Exercism\java\dnd-character\src\test\java\DnDCharacterTest.java:124: error: cannot find symbol
                    10 + character.modifier(character.getConstitution()));
                                                     ^
  symbol:   method getConstitution()
  location: variable character of type DnDCharacter
12 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
2 actionable tasks: 1 executed, 1 up-to-date
The terminal process terminated with exit code: 1
```

--- a/exercises/dnd-character/src/main/java/DnDCharacter.java
+++ b/exercises/dnd-character/src/main/java/DnDCharacter.java
@@ -8,4 +8,32 @@ class DnDCharacter {
         throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
     }
 
+    int getStrength() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getDexterity() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getConstitution() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getIntelligence() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getWisdom() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getCharisma() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getHitpoints() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
 }

